src/code/z_play.c

```diff
@@ -1265,10 +1265,10 @@ void Play_DrawMain(PlayState* this) {
                     if (this->skyboxId && !this->envCtx.skyboxDisabled) {
```

Comment:
```suggestion
                    if (((u32)this->skyboxId != SKYBOX_NONE) && !this->envCtx.skyboxDisabled) {
```
Might need the `(u32)` cast trick for this, but you can try without first. Also Jenkins isn't happy about formatting :)

Reply:
Yes, I just noticed both of them and already pushed them, but thanks